<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trajectory</class>
 <widget class="QWidget" name="trajectory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>881</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
